src/features/form/containers/RepeatingGroupsEditContainer.tsx

```diff
@@ -1,18 +1,17 @@
 import React from 'react';
 
 import { Button, ButtonColor, ButtonVariant } from '@digdir/design-system-react';
-import { Grid, makeStyles } from '@material-ui/core';
+import { Grid } from '@material-ui/core';
 import { Back, Delete as DeleteIcon, Next } from '@navikt/ds-icons';
 import cn from 'classnames';
 
 import { useAppSelector } from 'src/common/hooks/useAppSelector';
 import { renderLayoutNode } from 'src/features/form/containers/Form';
+import css from 'src/features/form/containers/RepeatingGroup.module.css';
```

Comment:
What do you think about renaming CSS to classes? Because the "objects" contains a set of class names.
```suggestion
import classes from 'src/features/form/containers/RepeatingGroup.module.css';
```